pages/mod/_id/versions.vue

```diff
@@ -38,7 +38,7 @@
                 version.id
               "
             >
-              {{ version.name ? version.name : version.version_number }}
+              <b>{{ version.name ? version.name : version.version_number }}</b>
```

Comment:
Why use a bold element?

Reply:
oops, must have been testing things, and forgot to remove it. Will fix.